src/tests/integration_test/wallet_test.cljs

```diff
@@ -0,0 +1,63 @@
+(ns tests.integration-test.wallet-test
+  (:require
+    [cljs.test :refer [deftest is]]
+    [day8.re-frame.test :as rf-test]
+    legacy.status-im.events
+    [legacy.status-im.multiaccounts.logout.core :as logout]
+    legacy.status-im.subs.root
+    status-im.events
+    status-im.navigation.core
+    status-im.subs.root
+    [test-helpers.integration :as h]
+    [tests.integration-test.constants :as test-constants]
+    [utils.re-frame :as rf]
+    [utils.security.core :as security]))
+
+(defn get-default-account
+  [accounts]
+  (first (filter :wallet accounts)))
+
+(def send-amount "0.0001")
+
+(deftest wallet-send-test
+  (h/log-headline :wallet-send-test)
+  (rf-test/run-test-async
+   (h/with-app-initialized
+    (h/with-recovered-account
+     (rf-test/wait-for
+       [:wallet/get-wallet-token]
+       (let [accs            (rf/sub [:wallet/accounts])
+             default-account (get-default-account accs)
+             address         (:address default-account)]
+         (rf/dispatch [:wallet/navigate-to-account (:address default-account)])
+         (rf-test/wait-for [:wallet/navigate-to-account]
+           (rf/dispatch [:wallet/select-send-address
+                         {:address    address
+                          :token      true
+                          :receipient address
+                          :stack-id   :wallet-select-address}])
+           (let [filtered-tokens (rf/sub [:wallet/tokens-filtered "eth"])]
+             (rf/dispatch [:wallet/send-select-token
+                           {:token    (first filtered-tokens)
+                            :stack-id :wallet-select-asset}])
+             (rf-test/wait-for
+               [:wallet/clean-suggested-routes]
+               (rf/dispatch
+                [:wallet/get-suggested-routes {:amount send-amount}])
+               (rf-test/wait-for
+                 [:wallet/suggested-routes-success]
+                 (let [route (rf/sub [:wallet/wallet-send-route])]
+                   (is (true? (some? route)))
```

Comment:
I think we don't need that `true?`

Reply:
this is an early break on the test if no route is found. I can remove but I think it adds value 🤔

Reply:
> but I think it adds value

If it's intentional it's ok :+1: 

> this is an early break on the test if no route is found

I'm not getting it, could you please expand more on the "early break"?

I think it doesn't matter if one check (call to `is`) fails, the rest of the test is still executed :thinking:

Reply:
ah true, I still think it's worth checkn the route was found, it will help in the CI to know if it failed because of no route found!